src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,157 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private static final String INDEX_OUT_OF_BOUNDS_MESSAGE = "Index %d "
+            + "does not exist in this LinkedList!";
+    private static final int DEFAULT_DIVISOR = 2;
+    private static final int INDEX_STOPPING_FROM_TAIL = 1;
+    private static final int FIRST_ELEMENT_INDEX = 0;
+    private static final int EMPTY_LIST_VALUE = 0;
+    private Node<T> head;
+    private Node<T> tail;
+    private Node<T> current;
```

Comment:
I think it can be local variable.

Reply:
We use head, tail and current in few methods, is it necessary to make them local? Or you just noticed about empty list value?